### DISCUSSION TITLE
Welcome to CALCULATOR-WEBSITE Discussions!

### DISCUSSION THREAD

**Discussion by YoshithaRathnayake** [YoshithaRathnayake/CALCULATOR-WEBSITE#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lilysophie0121-jpg** [comment]
Hieno tapa jakaa ideoita ja kysymyksiä laskureista! Samalla tavalla kuin keskusteluyhteisössä voi oppia uutta, voit seurata ja laskea omia tuloja helposti meidän [palkka](https://parasprosenttilaskuri.fi/palkkalaskuri/) -työkalulla.
